pkg/kapp/clusterapply/change_set_view.go

```diff
@@ -48,3 +53,55 @@ func (v *ChangeSetView) Print(ui ui.UI) {
 func (v *ChangeSetView) Summary() string {
 	return v.changesView.Summary() // assumes Print was used before
 }
+
+func (v ChangeSetView) PrintCompleteYamlToBeApplied(ui ui.UI, conf ctlconf.Conf) error {
```

Comment:
`ChangeSetView` already has `ui` and `maskRules`. 
Also, I am thinking that we should move this to existing `Print` method present for ChangeSetView.